.github/workflows/integrate-cluster-cmd.yaml

```diff
@@ -60,7 +60,8 @@ jobs:
         run: |
           # should not use -it
           # ref: https://stackoverflow.com/questions/43099116/error-the-input-device-is-not-a-tty
-          docker exec tiup-cluster-control bash /tiup-cluster/tests/tiup-cluster/run.sh ${{matrix.cases}}
+          docker exec tiup-cluster-n1 /bin/bash -c "apt install -y zsh && sed -i '/tidb/s/bash/zsh/g' /etc/passwd"
```

Comment:
It is not recommended to add this, because users use bash as the default shell in production env. We should make sure bash available first, the zsh is additional.

Reply:
+1